### DISCUSSION TITLE
How are Sunshine and  Games on Whales / wolf related?

### DISCUSSION THREAD

**Discussion by Mannshoch** [LizardByte/community#505] [Sunshine] [answered]
While looking into docker I found a Link to  games on whales. - now I'm a bit confused.

How are these both related?

**Comment by ReenigneArcher** [comment]
GoW uses Sunshine, but "Wolf" doesn't.

**Reply by Mannshoch** [reply]
more confusing

https://github.com/games-on-whales/wolf

Isn't game of whale what LizardByte is for sunshine?

**Reply by ReenigneArcher** [reply]
https://github.com/games-on-whales/gow

**Reply by ReenigneArcher** [reply]
> Isn't game of whale what LizardByte is for sunshine?

No ...

**Reply by ReenigneArcher** [reply, marked as answer]
If you want to use Wolf, you should connect with their community. https://github.com/games-on-whales/wolf

**Reply by Mannshoch** [reply]
No, is not planed. As long I do not understand Wolf and what it made better I use Sunshine
